.gitignore

```diff
@@ -4,3 +4,4 @@ dist
 npm-debug.log
 .cache
 samples/.vscode/settings.json
+**/build
```

Comment:
Actually, build folder has to be commited for `install from github` feature we have in a webapp